src/game_interpreter.cpp

```diff
@@ -2880,17 +3016,19 @@ bool Game_Interpreter::CommandKeyInputProc(lcf::rpg::EventCommand const& com) {
 				_keyinput.keys[Keys::eRight] = true;
 				_keyinput.keys[Keys::eUp] = true;
 			}
-			_keyinput.keys[Keys::eNumbers] = param_size > 5 ? com.parameters[5] != 0 : false;
-			_keyinput.keys[Keys::eOperators] = param_size > 6 ? com.parameters[6] != 0 : false;
-			_keyinput.time_variable = param_size > 7 ? com.parameters[7] : false;
-			_keyinput.timed = param_size > 8 ? com.parameters[8] != 0 : false;
+			_keyinput.keys[Keys::eNumbers] = check_key(5);
+			_keyinput.keys[Keys::eOperators] = check_key(6);
+			_keyinput.time_variable = param_size > 7 && com.parameters[7]; // not a bool
```

Comment:
This looks wrong.

Reply:
Was not sure how to handle this because this is the only argument that is not converted to a bool 😅

Reply:
IMO just like before, but false will become 0 anyways:
`param_size > 7 ? com.parameters[7] : 0;`

Reply:
Oooh now I see what you mean. Yeah this is wrong.

Reply:
Now I see the problem: This was incorrectly transformed by clang-tidy because the expression is ``param_size > 7 ? com.parameters[7] : false;``. This does not break the code here but the ``false`` must be actually a ``0``...

Reply:
(what I said earlier) ;)